src/nvim/api/private/helpers.c

```diff
@@ -1258,11 +1259,30 @@ Dictionary api_metadata(void)
     init_ui_event_metadata(&metadata);
     init_error_type_metadata(&metadata);
     init_type_metadata(&metadata);
+    if (with_docs) {
+      init_docs_metadata(&metadata);
+    }
```

Comment:
I have no idea why, but this doesn't work at all. Running the following Lua code:
```lua
local info = vim.api.nvim_get_api {
  docs = false
}

for k, _ in pairs(info) do
  print(k)
end
```
`info` should contain the docs and thus `docstrings` should be one of the keys printed, but there isn't? That makes no sense to me, because I did some logging in `nvim_get_api` that confirmed that `api_metadata` _is_ getting a `true` value. What am I missing?

Reply:
because of static initialization `static Dictionary metadata`

Reply:
i e you need to run `init_docs_metadata(&metadata)` and add to the dict _after_ `copy_dictionary`